pkg/minishift/registration/redhat.go

```diff
@@ -55,26 +55,7 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 		return err
```

Comment:
@praveenkumar if `subscription-manager version` command needs network access for this command to execute then we need to pass `-E` here.